Header header
sensor_msgs/Image coloredImage
geometry_msgs/PoseStamped POSE
geometry_msgs/Point[] LSDMap 
float32[] depth
float64 scale
